pkg/manager/internal/controllers/customresourcediscovery_controller.go

```diff
@@ -253,8 +254,10 @@ func (r *CustomResourceDiscoveryReconciler) reconcileCatapult(
 				Name: crDiscovery.Spec.ServiceCluster.Name,
 			},
 			WebhookStrategy: crDiscovery.Spec.WebhookStrategy,
+			LogLevel:        viper.GetInt("verbose"),
```

Comment:
This is too deep for the global variable lookup. It should be set in the controller's struct. 

And anyway, we discussed we're going to setup LogLevel in defaulting webhook if not otherwise stated.

Reply:
done